lte/gateway/c/core/oai/tasks/amf/include/amf_client_servicer.h

```diff
@@ -101,6 +101,11 @@ class AMFClientServicerBase {
                                      const std::string& ciphertext,
                                      const std::string& mac_tag,
                                      amf_ue_ngap_id_t ue_id);
+
+  virtual int n11_update_location_req(
+      const s6a_update_location_req_t* const ulr_p);
+
+  virtual bool set_smf_notification(SetSmNotificationContext& notify);
```

Comment:
```suggestion
  virtual bool set_smf_notification(const SetSmNotificationContext& notify);
```

Reply:
Making this change, required changing the API at `magma/lte/gateway/c/core/oai/lib/n11/SmfServiceClient.h`. Have made the changes. Comment addressed.